mscr-ui/src/modules/crosswalk-editor/mappings-accordion/index.tsx

```diff
@@ -60,6 +60,9 @@ import DeleteOutlineIcon from '@mui/icons-material/DeleteOutline';
 import styled from 'styled-components';
 import ArrowRightIcon from '@mui/icons-material/ArrowRight';
 import FunctionTooltipBox from "@app/modules/crosswalk-editor/mappings-accordion/function-tooltip-box";
+import ConfirmModal from "@app/common/components/confirmation-modal";
+import {setConfirmModalState} from "@app/common/components/actionmenu/actionmenu.slice";
+import {useStoreDispatch} from "@app/store";
```

Comment:
You ended up not using these last 2 imports, so let's not add them.

Reply:
Removing two imports on next commit.